docs/application/dotnet/guides/nui/xaml/ui-component.md

```diff
@@ -0,0 +1,116 @@
+# How to Define UI Component in XAML
+
+## XAML Object Elements
+Object element syntax always starts with an opening angle bracket (<). This is followed by the name of the type where you want to create an instance. After this, you can optionally declare attributes on the object element. To complete the object element tag, end with a forward slash and closing angle bracket in succession (/>). The following example shows how to write an object element syntax:
+
+``` xaml
+<TextLabel Text="TextLabel"/>
+```
+
+The `TextLabel` instance is created by calling the default constructor of the underlying type when parsing and loading the XAML.
+
+## Parent and Child
+The following example shows how to add a child view to the parent view:
+
+``` xaml
+<View>
+  <TextLabel Text="TextLabel"/>
+</View>
+```
+
+The `TextLabel` instance will be added to the `View` instance.
+
+## Attribute Syntax Properties
+An attribute syntax names the property that is being set in the attribute syntax, followed by the assignment operator (=). The value of an attribute is always specified as a string that is contained within the quotation marks. For example, the following markup creates a `TextLabel` that has a red text and a blue background in addition to the display text specified as `TextLabel`.
+
+``` xaml
+<TextLabel TextColor="Red" BackgroundColor="Blue" Text="TextLabel" />
+```
+
+In the preceding sample, we can use the string **Red** and **Blue** to set color, because there is a [TypeConverter](./TypeConverter.md).
```

Comment:
please verify the link:

Reference: https://github.com/Samsung/tizen-docs/pull/805#discussion_r320111243